### PR TITLE
Time stamps and concurrency analyses + inspection

### PR DESCRIPTION
**Pull request by mturilli**
This PR documents how to perform time stamps analyses with RA.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -51,11 +51,12 @@ RA supports the development and experimental analysis of the
...
    installation.rst
-   inspection.rst
+   plotting.rst
+   inspection.ipynb
    duration.ipynb
-   concurrency.rst
+   timestamps.ipynb
    utilization.ipynb
-   plotting.rst
+   concurrency.rst
```
Comment (lee212) [inline docs/source/index.rst:59]: Is this meant `concurrency.ipynb`?

File: docs/source/duration.ipynb [hunk 2]
```diff
@@ -74,8 +74,10 @@
    "source": [
     "Usually, it is useful to record the stack used for the analysis. \n",
     "\n",
-    "<div class=\"alert alert-block alert-info\">\n",
-    "    <b>Note:</b> The stack used for the analysis might be different from the stack used to crete the session to analyze. Usually, the two stack have to have the same major release number in order to be compatible.\n",
+    "<div class=\"alert alert-info\">\n",
+    "    \n",
+    "__Note:__ The stack used for the analysis might be different from the stack used to crete the session to analyze. Usually, the two stack have to have the same major release number in order to be compatible.\n",
```
Comment (lee212) [inline docs/source/duration.ipynb:79]: `crete` -> `create`, typo

File: docs/source/duration.ipynb [hunk 3]
```diff
@@ -139,22 +141,24 @@
...
+    "- __u_agent_schedule__: the amount of time taken to schedule of one or more tasks.\n",
+    "- __u_agent_t_pre_execute__: the amount of time taken to execute the ``pre_exec`` of one or more tasks.\n",
+    "- __u_agent_t_execute__: the amount of time taken to execute the executable of one or more tasks.\n",
+    "- __u_agent_t_stage_out__: the amount of time taken to stage the output data of one or more tasks.\n",
     "\n",
     "## Arbitrary Durations\n",
     "\n",
-    "RA enables the **arbitrary** definition of durations. What duration you need, depends on why you need a certain measure. For example, given an experiment to charaterize the performance of one of RP executors, it might be useful to measure the amount of time spent by each task in the Executor component. \n",
+    "RA enables the __arbitrary__ definition of durations. What duration you need, depends on why you need a certain measure. For example, given an experiment to charaterize the performance of one of RP executors, it might be useful to measure the amount of time spent by each task in the Executor component. \n",
```
Comment (lee212) [inline docs/source/duration.ipynb:156]: another typo, `charaterize` to `characterize`

**Review by lee212 — APPROVED** [review]
LGTM, and thanks for this PR! it will help many new users to try RA including myself.

File: docs/source/index.rst (hunk 1 above)
Comment (mturilli) [reply docs/source/index.rst:59]: Done, thanks!

File: docs/source/duration.ipynb (hunk 2 above)
Comment (mturilli) [reply docs/source/duration.ipynb:79]: Well spotted.

File: docs/source/duration.ipynb (hunk 3 above)
Comment (mturilli) [reply docs/source/duration.ipynb:156]: Thanks.

**Pull request merged**